robottelo/hosts.py

```diff
@@ -479,7 +509,27 @@ def execute_foreman_scap_client(self, policy_id=None):
         if result.status != 0:
             raise ContentHostError('Failed to execute foreman_scap_client run.')
 
-    def configure_rhai_client(self, activation_key, org, rhel_distro, register=True):
+    def configure_rex(self, satellite, org, subnet_id=None, by_ip=True):
+        """Setup a VM host for remote execution.
+
+        :param Satellite satellite: a hosts.Satellite instance
+        :param str org: The organization entity, label attr is used
+        :param int subnet: (Optional) Nailgun subnet entity id, to be used by the vm_client host.
+        :param bool by_ip: Whether remote execution will use ip or host name to access server.
+        """
+        self.install_katello_ca(satellite)
+        self.register_contenthost(org.label, lce='Library')
+        assert self.subscribed
+        add_remote_execution_ssh_key(self.ip_addr)
+        if subnet_id is not None:
+            Host.update({'name': self.hostname, 'subnet-id': subnet_id})
+        if by_ip:
+            # connect to host by ip
+            Host.set_parameter(
+                {'host': self.hostname, 'name': 'remote_execution_connect_by_ip', 'value': 'True'}
+            )
+
+    def configure_rhai_client(self, satellite, activation_key, org, rhel_distro, register=True):
```

Comment:
same comment about not forcing a registration per call of this method

Reply:
same comment about default keeping current behavior.